state/account.go

```diff
@@ -25,20 +25,25 @@ var (
 
 // Account is the canonical representation of an account.
 type Account struct {
-	// 0 is reserved from actions in genesis block and coinbase transfers nonces
+	// for Type 0, nonce 0 is reserved from actions in genesis block and coinbase transfers nonces
 	// other actions' nonces start from 1
-	Nonce        uint64
+	nonce        uint64
```

Comment:
i think it's OK to leave it as it, or is there a particular reason to make `nonce` unexported?